core/src/main/java/hudson/util/XStream2.java

```diff
@@ -257,8 +258,9 @@ private void init() {
         registerConverter(new AssociatedConverterImpl(this), -10);
 
         registerConverter(new BlacklistedTypesConverter(), PRIORITY_VERY_HIGH); // SECURITY-247 defense
+        addPermission(AnyTypePermission.ANY); // covered by JEP-200, avoid securityWarningGiven
```

Comment:
May need to be adjusted for XStream 1.5.x: https://github.com/x-stream/xstream/issues/207

Reply:
No. This is the same for XStream 1.4.x and 1.5.x. With this line you overwrite any default initialization and you are initializing the XStream Security Framework with a blacklist without any entries i.e. you allow anything. Your BlacklistedTypesConverter is now responsibly to cover any of the CVEs.

Reply:
In our case that is fine since we are actually using a whitelist, despite the name, so this line merely ensures that the current behavior is maintained and that XStream does not print a warning to stderr. https://jenkins.io/jep/200 FTR